lib/main_containers.pm

```diff
@@ -89,7 +89,8 @@ sub load_host_tests_podman {
         load_image_test($run_args) unless is_public_cloud || is_alp;
         load_3rd_party_image_test($run_args);
         loadtest 'containers/podman_pods';
-        loadtest 'containers/podman_network';
+        loadtest('containers/podman_network', name => 'podman_network_cni');
```

Comment:
I guess we can leave with the default name... maybe we can tweak `podman_network` in another PR to run it with netavark backend, although the operations you do with networks in this test module are kinda covering the use cases in `podman_network`, so maybe not needed eventually.

Reply:
Yeah, that was my thinking when dropping the second schedule of the module. Sure, I can revert the change over here, however I still find it useful that the test module also reflects which backend is being tested.

Reply:
Ok, in that case I would change the module name :D. But ok.